runtime/oti/j9port.h

```diff
@@ -759,6 +759,12 @@ typedef struct J9CacheInfoQuery {
 #define J9PORT_SLOPEN_NO_LOOKUP_MSG_FOR_NOT_FOUND OMRPORT_SLOPEN_NO_LOOKUP_MSG_FOR_NOT_FOUND
 #define J9PORT_SLOPEN_OPEN_EXECUTABLE OMRPORT_SLOPEN_OPEN_EXECUTABLE
 
+#if defined(J9ZOS39064)
+#define J9PORT_SLOPEN_ATTEMPT_31BIT_OPEN OMRPORT_SLOPEN_ATTEMPT_31BIT_OPEN
```

Comment:
This isn't required, the code can just use the OMR constants directly.

Reply:
Oh, okay.  I was just following the examples above. :)

Reply:
We didn't want to change all the code when OMR was split, but anything new can use OMR directly.

Reply:
I dropped these 3 macros in j9port.h and updated the couple places where I used the newly added J9PORT macros in a16bb7a2b122857ac766df05c80832235b8ea4d3,  dbaa1783c862d2d927895fef6ae6a98939ef8496 and 14598851209ed963aff5ac2c8501b7a662c40332.